src/fciqmc.jl

```diff
@@ -192,4 +192,20 @@ end
 function advance!(algorithm, report, state::ReplicaState, replica::SpectralState{1})
     return advance!(algorithm, report, state, only(replica.single_states))
 end
-# TODO: add advance! for SpectralState{N} where N > 1
+
+function advance!(algorithm, report, state::ReplicaState, replica::SpectralState{N, <:Any, GramSchmidt{N}}) where {N}
+    proceed = true
+    if state.step[] % replica.spectral_strategy.orthogonalization_interval == 0
+        for i in 1:N
+            for j in 1:i-1
+                u = replica[i].v
+                v = replica[j].v
+                add!(u, v, -dot(u, v) / dot(v, v))
```

Comment:
```suggestion
                add!(u, v, -dot(u, v) / norm(v)^2)
```
`norm` is much faster to evaluate the `dot` for `DVecs`